### ISSUE TITLE
Custom Towers Not Working On Normal Mode

### ISSUE THREAD

**Issue by CoolModder** [FortRise/FortRise#10]
Crashes upon entering a custom workshop level.
Error:
```
TowerFall Error Log
==========================================
>>>>>

Ver 1.3.3.3
10/6/2023 9:37:11 PM
Debug Flags: 0000 - 0 - Player: Normal
System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at TowerFall.TreasureSpawner..ctor(Session session, Int32[] mask, Single arrowChance, Boolean arrowShuffle)
   at TowerFall.CustomLevelSystem.GetTreasureSpawner(Session session)
   at DMD<TowerFall.Session::StartGame>(Session this)
   at DMD<TowerFall.MapScene::StartSession>(MapScene this)
   at Trampoline<TowerFall.MapScene::StartSession>?18473894(MapScene )
   at EightPlayerMod.MapButtonPatch.MapSceneStartSession_patch(orig_StartSession orig, MapScene self)
   at Hook<TowerFall.MapScene::StartSession>?53451813(MapScene )
   at Monocle.Alarm.Update()
   at Monocle.Entity.Update()
   at Monocle.Layer.Update()
   at Monocle.Scene.Update()
   at TowerFall.MapScene.orig_Update()
   at TowerFall.MapScene.Update()
   at Monocle.Engine.Update(GameTime gameTime)
   at TowerFall.TFGame.orig_Update(GameTime gameTime)
   at TowerFall.TFGame.Update(GameTime gameTime)
   at Microsoft.Xna.Framework.Game.Tick()
   at Microsoft.Xna.Framework.Game.RunLoop()
   at Microsoft.Xna.Framework.Game.Run()
   at TowerFall.TFGame.orig_Main(String[] args)

```
Tower: [ShootingRange.zip](https://github.com/Terria-K/WiderSetMod/files/12836296/ShootingRange.zip)

**Comment by Terria-K** [comment]
Probably the reason why it did this because the treasure I've implemented resized the array of all treasures. Workshop levels used numbers to define the treasure rates and its fixed to vanilla treasures if the levels are made in non-FortRise levels. I'll find a way to fix this.

**Comment by Terria-K** [comment]
Fixed in 4.4.3. Wait for the update.
